digitalpublicgoods/vaccineledger.json

```diff
@@ -0,0 +1,102 @@
+      "REST"
+    ],
+    "evidenceStandardSupport": [
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/Standards"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Principles of Digital Development ",
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/Standards"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "Our Risk Management strategy includes  Include risk management in our projects (Guidelines / Checklist/ Templates) Communicate risks to other stakeholders. Prioritize risks. Analyze risks with metrics. Implement risk responses as early as possible to avoid aggravating issues. Track them down regularly for compliance."
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/VaccineLedger_-_Data_Confidentiality.pdf"
+      ],
+      "thirdPartyDataSharing": "No",
+      "dataSharingCircumstances": [
+        ""
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "User information like phone and email are not stored - https://gitlab.com/statwig-public/theledger/-/blob/050a19d2c92a2836bfb9483941e5219e5b90be62/docs/VaccineLedger_-_Data_Confidentiality.pdf"
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "No",
+      "type": "",
+      "contentFilter": "",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "",
+      "illegalContentDetectionMechanism": ""
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "Yes",
+      "addressSafetySecurityUnderageUsers": "Yes",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [
+        "The users on our platform are supply chain managers, operational teams, logistics partners and so on. The user onboarding in each of these different types of users is carried out by a two step process:",
+        "- Step 1: User registers using name, official emails and/or phone numbers for verification and other identification details when needed. ",
+        "- Step 2: An admin reviews and approves the users."
+      ],
+      "griefAbuseHarassmentProtection": "Yes",
+      "harassmentProtectionSteps": [
+        "The safety and wellbeing of those we work with is of paramount importance to us. This includes staff, and the people  we work  with. We  have a  zero-tolerance  approach towards all forms of unethical behavior. We  also  have  a  zero  tolerance  towards  bribes,  facilitation  payments,  tax  evasion  and  money laundering. We will do all we can to prevent financing or the support of terrorist organisations and will not appoint or employ anyone that could reasonably be suspected to be connected to a terrorist group. We will abide by all the Laws in all matters including those related to modern slavery."
```

Comment:
It isn't mentioned here, but looks like some of their open source repositories also have a [code of conduct](https://gitlab.com/statwig-public/theledger/-/blob/development/CODE_OF_CONDUCT.md). It would be worth mentioning or linking here too as protection for contributors from harassment and abuse.

Reply:
Resolved in commit c002b88